examples/notebooks/beam-ml/run_inference_tensorflow.ipynb

```diff
@@ -313,14 +326,34 @@
       "metadata": {
         "id": "2JbE7WkGcAkK"
       },
-      "execution_count": 8,
+      "execution_count": 18,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Instead of saving the entire model, you can just save the model weights for inference. This is slightly lightweight than saving and loading the entire model. However, you need to pass the function to build TensorFlow model to the `TFModelHandlerNumpy` / `TFModelHandlerTensor` class along with `ModelType.SAVED_WEIGHTS`."
```

Comment:
```suggestion
        "Instead of saving the entire model, you can save the model weights for inference. This method might be simpler than saving and loading the entire model. However, you need to pass the function to build the TensorFlow model to the `TFModelHandlerNumpy` or `TFModelHandlerTensor` class, as well as `ModelType.SAVED_WEIGHTS`."
```

A couple things are unclear to me from this sentence:
1. Should this be "or" or "and": `TFModelHandlerNumpy` or `TFModelHandlerTensor` class
2. Is `ModelType.SAVED_WEIGHTS` also being passed to `TFModelHandlerNumpy` or `TFModelHandlerTensor`? We should make whatever is happening with `ModelType.SAVED_WEIGHTS` a new sentence and be clear about what we're doing with it.